app/src/main/java/app/gamenative/data/Game.kt

```diff
@@ -0,0 +1,18 @@
+package app.gamenative.data
+
+import app.gamenative.enums.AppType
+
+/**
+ * Unified interface for all game types (Steam, GOG, etc.)
+ */
+interface Game {
+    val id: String
+    val name: String
+    val source: GameSource
+    val isInstalled: Boolean
+    val isShared: Boolean
+    val iconUrl: String
+    val appType: AppType
+
+    fun toLibraryItem(index: Int): LibraryItem
+}
```

Comment:
isn't this the same as just having a libraryitem?

Why is isShared and iconUrl needed here? 

Don't we just need id, name, source and potentially isInstalled?

Reply:
This class had a different use. But it has since been removed.